ui/pages/Search/selectors.js

```diff
@@ -58,6 +58,21 @@ export const getProjectFamilies = (params, familiesByGuid, familiesByProjectGuid
   return null
 }
 
+export const getMultiProjectFamilies = createSelector(
+  (state, props) => props.match.params,
+  params => ({
```

Comment:
It does nothing with the state. Is it helpful to be a selector?

Reply:
yes because it memoizes the function so its not recomputed every time